compiler/rustc_hir_analysis/src/collect/predicates_of.rs

```diff
@@ -340,7 +340,7 @@ fn gather_explicit_predicates_of(tcx: TyCtxt<'_>, def_id: LocalDefId) -> ty::Gen
         };
         debug!(?lifetimes);
 
-        compute_bidirectional_outlives_predicates(tcx, &generics.params, &mut predicates);
+        compute_bidirectional_outlives_predicates(tcx, &generics.params, &mut vec![]);
```

Comment:
uh, please add a comment as to why it's ok that we're dropping these bounds on the floor. Because afaict, modulo some checks, this function call is now a no-op.

Reply:
lol, no, I left that in due to testing.

it's fine because i don't actually think we actually USE these predicates anywhere (i'm pretty sure we never call predicates_of on an opaque??), but i'm just an idiot and forgot to put this back.

Reply:
there 🤦 fixed